packages/api/src/queries/get-bewoner-aantal.ts

```diff
@@ -1,7 +1,9 @@
 import { gql } from "@apollo/client";
 
 export const QUERY_GET_BEWONER_AANTAL = gql`
-  query GetBewonersAantal {
-    getBewonersAantal
+  query GetBewonersAantal($adresseerbaarObjectIdentificatie: String!) {
```

Comment:
Gebruik je deze query nog op een andere plek? Anders verwijderen?

Reply:
Nee, dus zou verwijderd kunnen worden, maar kan ook blijven staan voor het geval iemand die query in de implementatie wilt blijven gebruiken.

Reply:
Oké, de query is nu verwijderd.